README.md

```diff
@@ -105,6 +105,7 @@ function hover(L=2^5;Re=250,U=1,amp=0,thk=1+√2)
     Simulation((6L+2,6L+2),zeros(2),L;U,ν,body,ϵ=0.5)
 end
 ```
+<!---  Would be cool to see how you generated the gif, I had a hard time getting any visuals out of this simulation --->
```

Comment:
The last three lines in [the block example](https://github.com/weymouth/WaterLily.jl/blob/master/examples/TwoD_block.jl) are commented out _uses_ of the `block` function. The last line generates this gif.

Reply:
I see, would still be cool if you link the file below the gif, something like [file](gif_generating_file) so everyone can understand which kind of field you plot there. I mean the gifs are very flashy 😄